environments/infrastructure/configuration.yml

```diff
@@ -7,5 +7,12 @@ netbox_tag: 'v4.1.4'
 netbox_userid: 'unit:root'
 netbox_plugins_defaults:
   - netbox_initializers
+netbox_plugins_extra:
+  - netbox_config_diff
+  - netbox_topology_views
+netbox_plugins_config:
+  netbox_config_diff:
+    USERNAME: "admin"
+    PASSWORD: "YourPaSsWoRd"
```

Comment:
To avoid false positives of the Github Credential Checkers we should encrypt the password with Ansible-Vault, even if this does not make much technical sense if it is still the default password.

Reply:
Agree 84374362993efeacf05417c0e1eb40df7c2703d7